service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -86,6 +87,9 @@
     @MockBean
     private CoreCaseDataService coreCaseDataService;
 
+    @MockBean
+    private InboxLookupService inboxLookupService;
```

Comment:
Did we mean to add just this mockBean? No further updates are added to this file.